api/metadata_handler_medium_test.go

```diff
@@ -0,0 +1,133 @@
+// +build medium
+
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"testing"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/web-platform-tests/wpt.fyi/api/query"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+
+func TestFilterMetadataHanlder_Success(t *testing.T) {
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		http.ServeFile(w, r, "../shared/metadata_testdata/gzip_testfile.tar.gz")
+	}
+	server := httptest.NewServer(http.HandlerFunc(handler))
+	defer server.Close()
+
+	r := httptest.NewRequest("GET", "/abd/api/metadata?product=chrome&product=safari", nil)
+	w := httptest.NewRecorder()
+	client := &http.Client{}
```

Comment:
```suggestion
	client := server.Client()
```

The benefit of this is that this test client will suppress any TLS errors with the local test server.

Reply:
Aha thanks!